### PR TITLE
adding bank account model and tests

### PR DESCRIPTION
**Pull request by sarias-eb**
We are adding fields in taxpayer form, in this case we need bank account fields for arg

### CODE REVIEW

File: supplier_app/models.py [hunk 1]
```diff
@@ -401,6 +401,31 @@ def __str__(self):
...
+        verbose_name=_("Payment type")
+    )
+    account_type = models.CharField(
+        max_length=60,
+        verbose_name=_("Account type"),
+    )
+    account_type = models.CharField(
+        max_length=60,
+        verbose_name=_("Account type"),
+    )
```
Comment (ricardocaif-eb) [inline supplier_app/models.py:416]: Repeated field

Comment (sarias-eb) [reply supplier_app/models.py]: Resolved! thanks :)

File: supplier_app/models.py [hunk 2]
```diff
@@ -401,6 +406,27 @@ def __str__(self):
         )
 
 
+class BankAccountArgentina(models.Model):
```
Comment (nicolasnunez-eb) [inline supplier_app/models.py:409]: If we are making a specific bank account for Argentina we should move all the specific fields like `bank_cbu_file` to this model.

File: supplier_app/models.py [hunk 3]
```diff
@@ -401,6 +406,27 @@ def __str__(self):
...
+    )
+    account_type = models.IntegerField(
+        choices=get_account_type_info_choices(),
+        verbose_name=_('Account type')
+    )
+    beneficiary = models.CharField(
+        max_length=60,
+        verbose_name=_("Beneficiary"),
+    )
+    bank_account = models.ForeignKey(
```
Comment (nicolasnunez-eb) [inline supplier_app/models.py:422]: I think that this relation should be a OneToOne, or is there any reason on having multiple Argentinian bank accounts for one general bank account.

File: supplier_app/models.py (hunk 2 above)
Comment (ricardocaif-eb) [reply supplier_app/models.py:409]: bank_info is specific for Argentina as well

**Pull request closed without merging**